docs/generate-time-series/javascript.rst

```diff
@@ -0,0 +1,241 @@
+.. _gen-ts-javascript:
+
+==========================================
+Generate time series data using Javascript
+==========================================
+
+This tutorial will show you how to generate some :ref:`experimental time series
+data <gen-ts>` by consuming telemetry data from the `International Space Station`_
+using Javascript, specifically `Node.js`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+CrateDB must be :ref:`installed and running <install-run>`.
+
+Make sure you're running up-to-date `Node`_.
+
+Also make sure you're running up-to-date `npm`_. To upgrade npm:
+
+.. code-block:: console
+
+    $ npm install -g npm@latest
+
+Use npm to install the `node-postgres`_ library:
+
+.. code-block:: console
+
+    $ npm install pg
+
+You will also be using the `Axios`_ library to make HTTP queries. Install the ``axios``
+library with npm:
+
+.. code-block:: console
+
+    $ npm install axios
+
+
+Both the ``axios`` and ``node-postgres`` libraries operate using `Promises`_. Promises are
+a way of encapsulating the eventual result of an asynchronous operation. Both ``axios``, when
+retrieving content from a URL, and ``node-postgres``, when connecting and querying a database,
+use Promises as a way of asynchronously doing those operations.
+
+If you are not familiar with asynchronous operations and Promises, Mozilla have written a
+`detailed guide`_ on the subject.
+
+Most of this tutorial is designed for Node's `interactive REPL mode`_ so that
+you can experiment with the commands as you see fit. Since both the libraries you will import
+use Promises, you should start ``node`` with support for the `await`_ operator:
+
+.. code-block:: console
+
+    $ node --experimental-repl-await
+
+
+Use Javascript to get to get ISS telemetry data
+===============================================
+
+You will be consuming telemetry data from `Open Notify`_, a service which provides
+a simple API to consume data from NASA. One of these data points is the current location
+of the International Space Station. The endpoint for these data can be found at
+`<http://api.open-notify.org/iss-now.json>`_.
+
+Start an interactive Node session using the ``node`` command. You can read the current
```

Comment:
I would drop the "using the `node` command" bit, because you've already detailed how to do this above. but, if you want to keep it, I would add the `--experimental-repl-await` again here

Reply:
new para at "You"